src/Sylius/Bundle/AdminBundle/Twig/ChannelExtension.php

```diff
@@ -30,12 +31,12 @@ public function __construct(ChannelRepositoryInterface $channelRepository)
     public function getFilters(): array
     {
         return [
-            new TwigFilter('sylius_channel_name', [$this, 'getChannelNameByCode']),
+            new TwigFilter('sylius_channel', [$this, 'getChannelByCode']),
```

Comment:
I'm not sure if it is not a BC break, I would be in favor of adding a new extension with this twig filter